src/features/cart/apis/CartItemsAPI.ts

```diff
@@ -0,0 +1,50 @@
+import { httpClient } from "../../../apis/httpClient";
+import { CartItem } from "../../../apis/types/response";
+
+const ERROR_MESSAGE = {
+  GET: "장바구니를 가져오는 데 실패했습니다.",
+  POST: "상품을 장바구니에 추가하는 데 실패했습니다.",
+  DELETE: "장바구니에서 상품을 삭제하는 데 실패했습니다.",
+  PATCH: "장바구니 상품 수량을 변경하는 데 실패했습니다.",
+};
+
+const ENDPOINT = "cart-items";
+
+export const CartItemsAPI = {
+  get: async (): Promise<CartItem[]> => {
+    const params = new URLSearchParams({
+      page: "0",
+      size: "50",
+      sort: "asc",
+    });
+
+    const response = await httpClient.get(`${ENDPOINT}?${params.toString()}`);
+    if (!response.ok) throw new Error(ERROR_MESSAGE.GET);
+
+    return await response.json();
+  },
```

Comment:
```suggestion
  get: async () => {
    const params = new URLSearchParams({
      page: "0",
      size: "50",
      sort: "asc",
    });

    const response = await httpClient.get<CartItem[]>(`${ENDPOINT}?${params.toString()}`);
    if (!response.ok) throw new Error(ERROR_MESSAGE.GET);

    return await response.json();
  },
```
이렇게 generic을 통해 타입을 지정할 수 있도록 httpClient를 정의해주면 어떨까요?